site/content/blog/getting-started-with-ytt-part-2.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Getting Started With ytt, Part 2"
+slug: getting-started-with-ytt-part-2
+date: 2022-11-13
+author: Varsha Munishwar
+excerpt: "Ever needed to make the same edits in multiple places in your kubernetes manifests? That can be error-prone. Learn how ytt can help you avoid misconfigurations." 
+image: /img/ytt.svg
+tags: ['ytt', 'ytt getting started', 'tutorials']
+---
+
+#### Welcome to the "Getting started with ytt" tutorial series!
+
+In the Part 2, we will cover the following:
+- Summary of [Part 1](getting-started-with-ytt-part-1/).
+- Dive into a slightly more involved scenario to solve a common problem in Kubernetes.
+- Learn why some set of labels are required to be in sync.
+- Introduce a ytt feature to avoid misconfigurations due to manual edits.
+
+#### Getting started with ytt - Part 2
+{{< youtube id="brOoKhVxedE" title="Getting started tutorial - Part 2" >}}
+
+
+**Note:**
+- The **key moments/timestamps** are available if you watch on youtube. Please click on "**more**" and "**view all**".
+- Here is the sample [deployment configuration](https://carvel.dev/ytt/#gist:https://gist.github.com/vmunishwar/db610648e999bebeb8743eb6eddd2d40) that is used in this tutorial to deploy the application on kubernetes cluster.
+- Visit the documentation for more information on [functions in ytt](https://carvel.dev/ytt/docs/v0.43.0/how-to-modularize/#functions).
+- In this tutorial, we are deploying the application on kubernetes using [Carvel's](https://carvel.dev/) deployment tool called kapp. If you are interested to know more here is the [link]( https://carvel.dev/kapp/). 
+
+
+Happy Templating :)
+
```

Comment:
Will there be any more ytt tutorials? If so, maybe end with "we will continue this series of ytt tutorials with topics covering x, x, and more as the tool develops. Stay tuned for when we post the next part in the coming months." ??

Reply:
As of now, we are not sure about the timeline for the next parts of this series. So, we are skipping this part. Thank you for the review and your suggestions on this PR

Reply:
Maybe we could end it with: 
If you enjoyed this series of tutorials, let us know if there are other Carvel topics, either ytt-specific or any of our other tools (imgpkg, kapp, kapp-controller, kbld, secretgen-controller, vendir, ytt), that you would like to see as a tutorial by reaching out to us in either in one of the channels listed in the section below, or on Twitter [@carvel_dev](https://twitter.com/carvel_dev) or [emailing us](mailto:carvel-dev@googlegroups.com) with your suggestion.

Reply:
Sounds good. I will add this note. Thank you